packages/manager-components/src/components/card/card.component.tsx

```diff
@@ -34,35 +34,46 @@ export interface CardProps {
     category: string;
   };
   badges?: Badge[];
-  trackingLabel?: string;
+  dataTrackingPath?: string;
+  dataTrackingValue?: string;
   hoverable?: boolean;
 }
 
-export const Card = ({
+export const Card: React.FC<CardProps> = ({
   href,
   isExternalHref,
   img,
   badges,
   texts,
-  trackingLabel,
+  dataTrackingPath,
+  dataTrackingValue,
   hoverable,
-}: CardProps) => {
+}) => {
   const { title, description, category } = texts;
   const { t } = useDynamicTranslation('card');
+  const { trackClick } = useOvhTracking();
 
   return (
     <a
       target={OdsHTMLAnchorElementTarget._blank}
       rel={OdsHTMLAnchorElementRel.noopener}
       href={href}
+      onClick={() => {
+        if (dataTrackingValue && dataTrackingPath) {
+          trackClick({
```

Comment:
its better to let free this attribute onClick, if user want to add an another behavior

Reply:
Yes there was no onClick function currently but as Quentin said we can still execute an onClick handler if we add one in the future